warehouse/integrations/clickhouse/clickhouse.go

```diff
@@ -770,7 +770,7 @@ func (ch *Clickhouse) schemaExists(ctx context.Context, schemaName string) (exis
 	sqlStatement := "SELECT count(*) FROM system.databases WHERE name = ?"
 	err = ch.DB.QueryRowContext(ctx, sqlStatement, schemaName).Scan(&count)
 	// ignore err if no results for query
-	if err == sql.ErrNoRows {
+	if errors.Is(err, sql.ErrNoRows) {
```

Comment:
Just wondering if we can move to `ExecContext` instead of `QueryRowContext` to avoid the no rows check?